covid19-etl/etl/ctp.py

```diff
@@ -324,8 +324,11 @@ def parse_row(self, row):
 
         for k, v in map_csv_fields.items():
             value = row[self.header_to_column[v]]
-            if value and value.lower() not in ["nan", "n/a"]:
-                summary_clinical[k] = int(value.replace(",", ""))
+            if value:
+                value = value.replace(",", "")
+                number = re.match(r"^([\s\d]+)$", value)
```

Comment:
why \s?

Reply:
try to match space sometime between numbers

Reply:
but it matches weird stuff like `0 1 2` 🤔